developer/python/CSharp_Partial_Classes_Helper.ipynb

```diff
@@ -0,0 +1,1249 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "ccbd3293",
```

Comment:
Commited a notebook that helped me write the partial classes, I'm quite sure I'll need it one day